receiver/filelogreceiver/README.md

```diff
@@ -19,35 +19,35 @@ Tails and parses logs from files.
 | `preserve_leading_whitespaces`      | `false`                              | Whether to preserve leading whitespaces.                                                                                                                                                                                                                        |
 | `preserve_trailing_whitespaces`     | `false`                              | Whether to preserve trailing whitespaces.                                                                                                                                                                                                                       |
 | `include_file_name`                 | `true`                               | Whether to add the file name as the attribute `log.file.name`.                                                                                                                                                                                                  |
 | `include_file_path`                 | `false`                              | Whether to add the file path as the attribute `log.file.path`.                                                                                                                                                                                                  |
 | `include_file_name_resolved`        | `false`                              | Whether to add the file name after symlinks resolution as the attribute `log.file.name_resolved`.                                                                                                                                                               |
 | `include_file_path_resolved`        | `false`                              | Whether to add the file path after symlinks resolution as the attribute `log.file.path_resolved`.                                                                                                                                                               |
-| `poll_interval`                     | 200ms                                | The duration between filesystem polls                                                                                                                                                                                                                           |
+| `poll_interval`                     | 200ms                                | The duration between filesystem polls. Takes `time.Duration` as value.                                                                                                                                                                                          |
```

Comment:
What do you mean `time.Duration`?  Assume the person using this component does knowledge about go. 
Can we maybe replace this by 
> The duration between filesytem pools. The duration can include the unit. e.g.: 200ms, 1s, 1m.

What happens if the unit is not specified?

Reply:
Sure I can make those changes. As for when it's not specified, it'll default to nanoseconds. I'll add that info to the doc as well